.gitignore

```diff
@@ -2,6 +2,8 @@ node_modules
 .vscode
 coverage
 lib
+dist
 examples
 logs
+server-production.js
```

Comment:
which build script generates this file, do you know @magicmatatjahu ? I couldn't find this while testing the PR.

Reply:
Thanks! I forgot about this file, previously I tried to use webpack but we have a lot of deps that don't work with bundling.